mods/ra/maps/allies-13/map.yaml

```diff
@@ -0,0 +1,1214 @@
+MapFormat: 11
```

Comment:
```suggestion
MapFormat: 12
```

map format should be bumped to 12. I also noticed a few of the other new missions have it at 11

Reply:
Those should be re-saved with the ingame map editor to update the version.